test/pummel/test-fs-readfile-tostring-fail.js

```diff
@@ -21,6 +21,8 @@ const stream = fs.createWriteStream(file, {
   flags: 'a'
 });
 
+stream.on('error', (err) => { assert.ifError(err); });
```

Comment:
Won't `assert.ifError()` always throw here?

Reply:
Yes, we don't expect write errors and I would rather it failed with the received error (if we don't get errors this listener won't be called). Would you prefer I just throw the `err`? (I originally had `common.mustNotCall()` as the listener but that wouldn't actually report the `err`.

Reply:
Yea, I think throwing the error directly makes more sense.

Reply:
done!